redirector.py

```diff
@@ -204,7 +222,7 @@ def main():
 
     settings = dict(
         stats=stats,
-        xsrf_cookies=True,
+        xsrf_cookies=False,
```

Comment:
@rgbkrk — Do we need this set to `True` if we aren't using cookies for authentication?